source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRChoiceSetViewDataSource.mm

```diff
@@ -15,6 +15,8 @@ @implementation ACRChoiceSetViewDataSource
     std::shared_ptr<ChoiceSetInput> _choiceSetDataSource;
     NSMutableDictionary *_userSelections;
     NSIndexPath *_lastSelectedIndexPath;
+    NSSet *_defaultValuesSet;
+    NSArray *_defaultVaulesArray;
```

Comment:
>Vaules [](start = 21, length = 6)

Values

Reply:
Thanks!

---
In reply to: [170704835](https://github.com/Microsoft/AdaptiveCards/pull/1167#discussion_r170704835) [](ancestors = 170704835)